ros/src/tl_detector/tl_detector.py

```diff
@@ -126,7 +129,8 @@ def process_image(self):
         self.image_cb_total_time += (t2 - t1)
         self.image_cb_counter += 1
         self.has_image = False
-        #print("Av. proc. time TL Detector: ", self.image_cb_total_time / self.image_cb_counter)
+        rospy.logdebug("TL Detector Current proc. time : %f s", (t2 - t1))
```

Comment:
I would make a difference between time logging and logging the results of the classification (unfortunately there is nothing above debug ...). Perhaps use info then for red/yellow/green/bad angle.